src/ICSharpCode.SharpZipLib/Tar/TarStringExtension.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.IO;
+
+namespace ICSharpCode.SharpZipLib.Tar
+{
+	internal static class TarStringExtension
+	{
+		public static string ClearTarPath(this string s)
```

Comment:
It's not obvious what `ClearTarPath` would do...
```suggestion
		public static string ToTarArchivePath(this string s)
```
This still doesn't explain *how* it's done, but it explains the purpose and what the expected outcome is.